xbmc/pvr/addons/PVRClients.h

```diff
@@ -639,6 +639,13 @@ namespace PVR
     void ConnectionStateChange(CPVRClient *client, std::string &strConnectionString, PVR_CONNECTION_STATE newState, std::string &strMessage);
 
   private:
+    /*!
+     * @brief Get a client's numeric ID for a given addon instance.
+     * @param addon The addon instance.
+     * @return The numeric ID matching the given addon instance, -1 on error.
+     */
+    int GetClientId(const ADDON::AddonPtr addon) const;
```

Comment:
Pass by const ref to avoid the smart pointer copy (and to bike shedders, yes, passing by ref is OK for smart pointers)

Reply:
Will change. Thanks.